app/controllers/admin/results_controller.rb

```diff
@@ -78,15 +80,16 @@ def search_by_users_ids(user_ids)
 
     def search_domain_names_result
       Result.joins(:auction)
-            .includes(:offer, :invoice)
-            .where('auctions.domain_name ILIKE ?', "%#{@origin}%")
+            .includes(:auction, :offer, :invoice)
```

Comment:
Why include the auction here, if it is already loaded at the SQL level through the joins method? It has already been merged with the results table, so there is no point in using it for preloading via include to avoid N + 1.

Reply:
:joins joins tables together in sql, :includes eager loads associations to avoid the n+1 problem (where one query is executed to retrieve the record and then one per association which is loaded). It will not work here if I remove one of these functions.